tools/mfassignr/macros.xml

```diff
@@ -0,0 +1,38 @@
+<macros>
+    <token name="@TOOL_VERSION@">1.0.3</token>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="@TOOL_VERSION@">r-mfassignr</requirement>
+        </requirements>
+    </xml>
+
+    <xml name="creator">
+        <creator>
+            <person
+                givenName="Zargham"
+                familyName="Ahmad"
+                url="https://github.com/zargham-ahmad"
+                identifier="0000-0002-6096-224X" />
+            <organization
+                url="https://www.recetox.muni.cz/"
+                email="GalaxyToolsDevelopmentandDeployment@space.muni.cz"
+                name="RECETOX MUNI" />
+        </creator>
+    </xml>
+
+    <xml name="isofiltr_param">
+        <param name="peaks" type="data" format="tabular" label="Input Peak Data"
+        help="The input data frame containing abundance and peak mass."/>
+        <param name="SN" type="float" value="0" label="Signal-to-Noise Ratio"
+        help="Sets the noise cut for the data; peaks below this value will not be evaluated."/>
+        <param name="Carbrat" type="float" value="60" label="Maximum 13C/12C Ratio"
```

Comment:
It fails when Max 13C/12C is 0, 1 or 2; 5 seems to be already fine, I haven't found any upper limit. Maybe we should limit the lower boundary?

Reply:
done in [5d269af](https://github.com/RECETOX/galaxytools/pull/565/commits/5d269afce6a1cdba26dc9813a12a9c5bffa94756)